forms/serializers/form.py

```diff
@@ -301,7 +301,13 @@ def get_information_checks(self, obj):
                     {
                         "id": information_check.id,
                         "name": information_check.name,
-                        "preparer": information_check.preparer,
+                        "preparer": {
```

Comment:
(Probably) just one more thing: the preparer needs to be checked for if it's None or it's back to 500 land. Can't verify if this works with an existing preparer because of that, but by reading the code, that part is probably fine now.